specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -0,0 +1,1912 @@
+          {
+            "$ref": "#/parameters/SavingsPlanOrderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/SavingsPlanIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SavingsPlanUpdateValidateRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request is completed.",
+            "schema": {
+              "$ref": "#/definitions/SavingsPlanValidateResponse"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.BillingBenefits/validate": {
+      "post": {
+        "description": "Validate savings plan purchase.",
+        "operationId": "SavingsPlanPurchase_Validate",
+        "tags": [
+          "Savings plan"
+        ],
+        "x-ms-examples": {
+          "SavingsPlanValidatePurchase": {
+            "$ref": "./examples/SavingsPlanValidatePurchase.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SavingsPlanPurchaseValidateRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request is completed.",
+            "schema": {
+              "$ref": "#/definitions/SavingsPlanValidateResponse"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.BillingBenefits/reservationOrderAliases/{reservationOrderAliasName}": {
+      "put": {
+        "description": "Create a reservation order alias.",
+        "operationId": "ReservationOrderAlias_Create",
+        "tags": [
+          "Reservation order aliases"
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "ReservationOrderAliasCreate": {
+            "$ref": "./examples/ReservationOrderAliasCreate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ReservationOrderAliasNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ReservationOrderAliasRequestParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

If you allow updates to the resource, you also need a 200

Reply:
Changed it to 200 to align with savingsPlanOrderAlias PUT call

Reply:
200 needs to be supported in addition to a 201